lib/src/services/gamepad.dart

```diff
@@ -0,0 +1,42 @@
+import "package:win32_gamepad/win32_gamepad.dart";
+
+import "service.dart";
+
+///A service to send the comtroller's activity
+///using win32_gamepad
+class GamepadService extends Service {
+  ///The connection to the controller
+  static final gamepad = Gamepad(0);
+  
+  @override
+  Future<void> init() async{ 
+    connect();
+  }
+
+  @override
+  Future<void> dispose() async{  
+    gamepad.state = GamepadState.disconnected();
+  } 
+
+  ///Check to see if the remote is connected
+  ///
+  ///returns true or false
+  Future<bool> connect() async{ 
+    if(gamepad.isConnected){
+      return true;
+    }else{
+      return false;
+    }
+  }
+
+  ///Return the current state of the remote
+  GamepadState getState() => gamepad.state; 
```

Comment:
Fun trick, you can do:
```suggestion
  /// The current state of the gamepad, its buttons, and its connection state. 
  GamepadState get state => gamepad.state; 
```

That way, callers can do `GamepadService.state` instead of `GamepadService.getState()`. Also, in general, it's good to start documentation by saying _what_ it actually returns -- the reader knows it's a function, so they already know it "returns" something.